src/index.js

```diff
@@ -0,0 +1,12 @@
+import './assets/global/reset.css';
+import './assets/main.scss';
+import Apple from './assets/images/apple.png';
+
+const app = document.querySelector('#app');
```

Comment:
DOM변수들은 $를 prefix로 붙여주면 일반 변수와 DOM 변수를 구분하기 좋습니다. 예전에 바닐라, jQuery로 개발할 때 관습이에요.

```suggestion
const $app = document.querySelector('#app');
```